copanier/models.py

```diff
@@ -503,13 +503,13 @@ def is_defined(cls):
 
     @classmethod
     def incoming(cls):
-        return sorted(
-            [d for d in cls.all() if d.is_foreseen], key=lambda d: d.order_before
-        )
+        incoming_deliveries = [d for d in cls.all() if d.is_foreseen]
+        return sorted(incoming_deliveries, key=lambda d: d.order_before)
```

Comment:
No feature change (hopefully). Just applying the same suggestion from the sorting being added below to this side as well.

Reply:
Perfect thanks :-)